PPM-383: Remove the unsupported 60Ghz band option from the selected bands

DanielEdriPurpl opened the pull request:
Remove from the persistent DB all references to the unsupported 60GHz band.
This includes the BML CLI, the DB, and anything in between.

itayx reviewed (approved):
Looks good! 👍

adam1985d reviewed (approved):
good PR - congratulations for your first PR to PRPL.

rmelotte commented:
There is something wrong with the autogenerated files:
```
$ git diff-index --exit-code HEAD
:100644 100644 6c5991d0ea83a14ff60c5e6e0c7fb48939eee768 0000000000000000000000000000000000000000 M	common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_common.h
```
They need to be generated again before this can be merged: https://gitlab.com/prpl-foundation/prplmesh/prplMesh/-/jobs/685221686
Make sure you don't run clang-format or any other formatter on it, we don't run them on the tlvf folder.

DanielEdriPurpl committed:
common: tlvf: remove the unsupported option of 60Ghz

The 60Ghz band is currently not supported, and shouldn't
be part of the available configuration of the selected bands.

Remove the 60Ghz from the eSelectedBands enum which is
used by the persistent DB.

PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>

DanielEdriPurpl committed:
bml: remove the unsupported 60Ghz option from selected bands

Remove the 60Ghz option from the BML_CLIENT_SELECTED_BANDS in both
bml_defs and bml_cli respectively.

PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
